ruby/test/ci/queue/redis/test_time_record_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class CI::Queue::Redis::TestTimeRecordTest < Minitest::Test
+  def setup
+    redis_url = "redis://#{ENV.fetch('REDIS_HOST', 'localhost')}/7"
+    redis = Redis.new(url: redis_url)
+    redis.flushdb
```

Comment:
Nitpick: this same pattern is used in multiple places, it would be nice to move this into a test helper module that any test can include if it needs a local Redis instance (more DRY). Feel free to ignore this for this PR.

Reply:
good idea, I'll add it in a separate PR.

Reply:
Can you create an issue for that and add it to the kanban board, so that this doesn't get lost?

Reply:
good idea, issue created: https://github.com/Shopify/ci-queue/issues/114